metadata-etl/src/test/java/metadata/etl/dataset/oracle/OracleMetadataEtlTest.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package metadata.etl.dataset.oracle;
+
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+
+public class OracleMetadataEtlTest {
```

Comment:
The test for wherehows is a joke. (I know it start from me...)
The standard way is to create a lot of mocks for all dependence call, e.g. the database connection call, the rest API call and so on, then use these mocks to test.
It would be very time consuming :( But it would be a longterm goal to achieved to let wherehows be a more mature and standard system.